src/status_im/contexts/wallet/swap/swap_confirmation/view.cljs

```diff
@@ -0,0 +1,200 @@
+    [quo/summary-tag
+     {:label               (:name account)
+      :type                :account
+      :emoji               (:emoji account)
+      :customization-color (:color account)}]]])
+
+(defn- summary-section
+  [{:keys [theme label title-accessibility-label amount token-symbol token-address network]}]
+  (let [network-values {(if (= network :mainnet) :ethereum network)
+                        {:amount amount :token-symbol token-symbol}}]
+    [rn/view
+     {:style {:padding-horizontal 20
+              :padding-bottom     16}}
+     [quo/text
+      {:size                :paragraph-2
+       :weight              :medium
+       :style               (style/section-label theme)
+       :accessibility-label title-accessibility-label}
+      label]
+     [quo/summary-info
+      {:type        :token
+       :networks?   true
+       :values      network-values
+       :token-props {:token   token-symbol
+                     :label   (str amount " " token-symbol)
+                     :address (address-utils/get-shortened-compressed-key token-address)
+                     :size    32}}]]))
+
+(defn- data-item
+  [{:keys [title subtitle loading?]}]
+  [quo/data-item
+   {:container-style style/detail-item
+    :blur?           false
+    :description     :default
+    :card?           false
+    :status          (if loading? :loading :default)
+    :size            :small
+    :title           title
+    :subtitle        subtitle}])
+
+(defn- transaction-details
+  [{:keys [estimated-time-min max-fees max-slippage loading-fees?]}]
+  [rn/view {:style style/details-container}
+   [:<>
+    [data-item
+     {:title    (i18n/label :t/est-time)
+      :subtitle (i18n/label :t/time-in-mins {:minutes (str estimated-time-min)})}]
+    [data-item
+     {:title    (i18n/label :t/max-fees)
+      :subtitle max-fees
+      :loading? loading-fees?}]
+    [data-item
+     {:title    (i18n/label :t/max-slippage)
+      :subtitle (str max-slippage "%")}]]])
+
+(defn footer
+  [{:keys [estimated-time-min native-currency-symbol max-slippage theme account-color provider
+           loading-fees?]}]
+  (let [native-token  (when native-currency-symbol
+                        (rf/sub [:wallet/token-by-symbol
+                                 native-currency-symbol]))
+        fee-formatted (rf/sub [:wallet/wallet-send-fee-fiat-formatted
+                               native-token])]
+    [:<>
+     [transaction-details
+      {:estimated-time-min estimated-time-min
+       :max-fees           fee-formatted
+       :max-slippage       max-slippage
+       :loading-fees?      loading-fees?
+       :theme              theme}]
+     [standard-auth/slide-button
+      {:size                :size-48
+       :track-text          (i18n/label :t/slide-to-swap)
+       :container-style     {:z-index 2}
+       :customization-color account-color
+       :disabled?           loading-fees?
+       :on-auth-success     #(js/alert "Not implemented")
```

Comment:
do we need the alert? 
I guess this is feature flagged anyway, so not such a big issue

Reply:
We don't need it, will remove